pkg/mimirtool/commands/alerts.go

```diff
@@ -94,6 +94,12 @@ func (a *AlertmanagerCommand) Register(app *kingpin.Application, envVars EnvVarN
 		cmd.Flag("id", "Grafana Mimir tenant ID; alternatively, set "+envVars.TenantID+". Used for X-Scope-OrgID HTTP header. Also used for basic auth if --user is not provided.").Envar(envVars.TenantID).Required().StringVar(&a.ClientConfig.ID)
 	}
 
+	prepareCmd := alertCmd.Command("prepare", "Prepare the Alertmanager tenant configuration for UTF-8.").Action(a.prepareConfig)
```

Comment:
Is this command intended to be something a user runs once to migrate configuration to UTF-8? Could we pick something more specific for the name, maybe `migrate-utf8` or something?

Reply:
> Is this command intended to be something a user runs once to migrate configuration to UTF-8?

Yes!

> Could we pick something more specific for the name, maybe migrate-utf8 or something?

Sure, sounds fine to me! Let's go with `migrate-utf8`.